The DocBook DTD must be 4.2 to validate with KDE 4.5.2.  In the upstream release 0.9.95
of kdiff3 some languages have index.docbook with the 4.1.2 DTD.  This patch changes
the version number in the DOCTYPE to 4.2 for these languages.


diff -Naur kdiff3-upstream-0.9.95/doc/de/index.docbook kdiff3-0.9.95/doc/de/index.docbook
--- kdiff3-upstream-0.9.95/doc/de/index.docbook	2010-10-18 22:38:40.043252907 -0430
+++ kdiff3-0.9.95/doc/de/index.docbook	2010-10-18 22:41:05.954329524 -0430
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY kdiff3 "<application
 >KDiff3</application
 >">
diff -Naur kdiff3-upstream-0.9.95/doc/fr/index.docbook kdiff3-0.9.95/doc/fr/index.docbook
--- kdiff3-upstream-0.9.95/doc/fr/index.docbook	2010-10-18 22:38:40.043252907 -0430
+++ kdiff3-0.9.95/doc/fr/index.docbook	2010-10-18 22:41:05.965408604 -0430
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY kdiff3 "<application
 >KDiff3</application
 >">
diff -Naur kdiff3-upstream-0.9.95/doc/it/index.docbook kdiff3-0.9.95/doc/it/index.docbook
--- kdiff3-upstream-0.9.95/doc/it/index.docbook	2010-10-18 22:38:40.044252773 -0430
+++ kdiff3-0.9.95/doc/it/index.docbook	2010-10-18 22:41:05.976408528 -0430
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY kdiff3 "<application
 >KDiff3</application
 >">
diff -Naur kdiff3-upstream-0.9.95/doc/nl/index.docbook kdiff3-0.9.95/doc/nl/index.docbook
--- kdiff3-upstream-0.9.95/doc/nl/index.docbook	2010-10-18 22:38:40.043252907 -0430
+++ kdiff3-0.9.95/doc/nl/index.docbook	2010-10-18 22:41:05.986408860 -0430
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
+<!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.2-Based Variant V1.1//EN" "dtd/kdex.dtd" [
   <!ENTITY kdiff3 "<application
 >KDiff3</application
 >">
